x-pack/platform/plugins/shared/search_connectors/public/api/index/create_api_index_api_logic.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { HttpSetup } from '@kbn/core/public';
+import { Actions, createApiLogic } from '../api_logic/create_api_logic';
+
+interface CreateApiIndexValue {
+  index: string;
+}
+
+export interface CreateApiIndexApiLogicArgs {
+  deleteExistingConnector?: boolean;
+  indexName: string;
+  language: string | null;
+  http?: HttpSetup;
+}
+
+export interface CreateApiIndexApiLogicResponse {
+  indexName: string;
+}
+
+export const createApiIndex = async ({
+  indexName,
+  language,
+  http,
+}: CreateApiIndexApiLogicArgs): Promise<CreateApiIndexApiLogicResponse | undefined> => {
+  const route = '/internal/search_connectors/indices';
```

Comment:
I don't see where this route exists.

Reply:
removed in [commit](https://github.com/elastic/kibana/pull/213509/commits/b03b5e02ae2b5f83413a34c9a22fa4e055c65a9c)